pandas/core/groupby/groupby.py

```diff
@@ -788,7 +785,7 @@ def __repr__(self) -> str:
 
     @final
     @property
-    def groups(self) -> dict[Hashable, np.ndarray]:
```

Comment:
is this an API change?  or just wrong in the status quo?

Reply:
status quo is wrong - main currently returns an Index